web/packages/teleterm/src/ui/Search/SearchBar.tsx

```diff
@@ -103,14 +103,12 @@ function SearchBar() {
         flex-shrink: 1;
         min-width: calc(${props => props.theme.space[7]}px * 2);
         height: 100%;
-        background: ${props => props.theme.colors.levels.sunkenSecondary};
-        border: 1px ${props => props.theme.colors.action.disabledBackground}
-          solid;
+        background: inherit;
+        border: 1px ${props => props.theme.colors.buttons.border.border} solid;
```

Comment:
I don't think this was introduced by this PR specifically and I don't think we necessarily need to fix this right now. I just want to point out something we should be aware of in the future.

We have five different elements in the top bar and almost every one of them behaves in a different way when it comes to how the border is handled when the element is active.


https://user-images.githubusercontent.com/27113/233377044-ffaf1cab-d325-4b9e-92e3-38698d7102d0.mov

Reply:
Definitely we should change the search input border in the cluster selector.
It would be good to get rid of the connections icon background, but the activity indicator looks weird then.

Reply:
> It would be good to get rid of the connections icon background, but the activity indicator looks weird then.

Yeah, and we're going to remove this whole UI element soon anyway.